app/helpers/layout_helper.rb

```diff
@@ -347,6 +347,18 @@ def fullscreen_button(element = "$(this).prev()")
   private
 
   def authorized_associations(associations)
-    associations.included_modules.include?(Authorizable) ? associations.authorized : associations
+    if associations.included_modules.include?(Authorizable)
+      if associations.respond_to?(:klass)
+        associations.authorized(authorized_assocations_permission_name(associations.klass), associations.klass)
+      else
+        associations.authorized(authorized_assocations_permission_name(associations), associations)
+      end
+    else
+      associations
+    end
+  end
+
+  def authorized_assocations_permission_name(klass)
+    "view_#{klass.to_s.downcase.pluralize}"
```

Comment:
I'm not sure this is correct for all cases, e.g. `ProvisioningTemplate.to_s.downcase.pluralize` is `provisioningtemplates`, but it should result in `view_provisioning_templates`.  I also wonder if there are plugins that add stuff to the taxonomy form and that might have more complex permission name rules?

I was trying to think if a functional or integration test covered whether the form displays if the permission is unknown, but I don't know that it would.  We might need a test to loop around taxonomy-related things and check this resolves to a known permission name too?